packages/fixtures/src/manuscripts.ts

```diff
@@ -24,7 +24,7 @@ export const createManuscriptResponse = (
   teamId: 'team-1',
   versions: [
     {
-      lifecycle: 'Draft manuscript (prior to preprint submission)',
+      lifecycle: 'Draft Manuscript(prior to Publication)',
```

Comment:
```suggestion
      lifecycle: 'Draft Manuscript (prior to Publication)',
```
Same comment that I made in the schema PR in a few different places